### ISSUE TITLE
Parquet output format

### ISSUE THREAD

**Issue by igorbrigadir** [DocNow/twarc-csv#29]
Instead of CSVs, append the parsed dataframes to parquet https://stackoverflow.com/a/47839247/11090908

**Comment by edsu** [comment]
Being able to output as parquet would be nice too--even if it's called twarc-csv :-)

**Comment by igorbrigadir** [comment]
Yeah I'm actually considering a different command as an alias, just for it to make semantic sense / good docs, so these would be the same:

```
twarc2 dataframe --output-format parquet input.json output.parquet

twarc2 csv --output-format parquet input.json output.parquet
```

But not sure how useful that is. It'll purely be an alias for a docs entry and for the command line.

**Comment by edsu** [comment]
I was going to say that pandas has many output formats. Since twarc-csv is use it it might not be hard to add parquet, pickle, hdf, sql, excel, json, html, feather, latex, stata, gbq, markdown :-)

**Comment by igorbrigadir** [comment]
Yeah I thought about adding an alias like 

```
twarc2 dataframe --output-format parquet input.json output.parquet
```

That's equivalent to 

```
twarc2 csv --output-format parquet input.json output.parquet
````

this way it would have nicer docs / more "semantic" command hopefully?

**Comment by igorbrigadir** [comment]
Yeah - still figuring out that part!
